raiden/blockchain_events_handler.py

```diff
@@ -171,6 +133,77 @@ def handle_channel_new(raiden: "RaidenService", event: Event):
     raiden.add_pending_greenlet(retry_connect)
 
 
+def create_light_channel(
+    block_hash,
+    block_number,
+    channel_identifier,
+    participant1,
+    participant2,
+    token_network_identifier,
+    transaction_hash,
+    raiden):
+    channel_state = create_channel(block_hash, block_number, channel_identifier, participant1, participant2, raiden,
+                                   token_network_identifier, transaction_hash)
+
+    partner_address = channel_state.partner_state.address
+
+    if ConnectionManager.BOOTSTRAP_ADDR != partner_address:
```

Comment:
this should be at create_channel

Reply:
if this is true, then i don't know if there is much point in having this function instead of calling `create_channel` directly